addons/mail/views/mail_templates.xml

```diff
@@ -259,6 +260,12 @@
             </xpath>
         </template>
 
+        <template id="qunit_mobile_suite" name="mail_mobile_tests" inherit_id="web.qunit_mobile_suite">
+            <xpath expr="//t[@t-set='head']" position="inside">
+                <script type="text/javascript" src="/mail/static/tests/owl/test_utils.js"></script>
```

Comment:
What changed that made this necessary? Did it just not work before? Are those tests not run on runbot?

Reply:
Discard this comment, I see it is for new `afterNextRender` in enterprise.